docs/source/getting_started/manufacturing/07_3d_visualization.ipynb

```
@@ -0,0 +1,379 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import open3d as o3d\n",
+    "import tifffile\n",
+    "\n",
+    "def convert_tiff_xyz_to_pcd_with_color(tiff_path, mask_path, pcd_path):\n",
+    "    \"\"\"\n",
+    "    Converts a .xyz TIFF file into a PCD file, adding color information from a mask.\n",
+    "    \n",
```

Ruff (0.12.2)

Loop control variable `dirnames` not used within loop body

Rename unused `dirnames` to `_dirnames`

(B007)